templates/form_builder/custom_tailwind_layout.html.twig

```diff
@@ -0,0 +1,72 @@
+{# @experimental in 5.3 #}

```

Comment:
Dlaczego zdecydowaliście się na nadpisywanie skórki tailwinda? Nie wystarczy użyć tej która jest przygotowana? Jeśli to kwestia klas na posczególnych polach, to chyba do tych pól wystarczy to nadać w FormTypie?

Reply:
Nie, bo nasz design language jest trochę inny niż domyślny styl dla tailwind